packages/bson-transpilers/test/yaml/native.yaml

```diff
@@ -859,6 +935,7 @@ tests:
           csharp: DateTime.Now
           object: "new Date()"
           ruby: Time.new.utc
+          rust: Utc::now()
```

Comment:
I think we'll need to use the `bson::DateTime::now()` here, since this example requires `chrono`, which we're trying to move away from.

Reply:
Updated.